test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -135,15 +136,15 @@ type holder struct {
 
 	t *testing.T
 
-	expectGVR       metav1.GroupVersionResource
-	expectGVK       schema.GroupVersionKind
-	expectOperation v1beta1.Operation
-	expectNamespace string
-	expectName      string
-	expectObject    bool
-	expectOldObject bool
-
-	recorded map[string]*v1beta1.AdmissionRequest
+	expectGVR              metav1.GroupVersionResource
+	expectGVK              schema.GroupVersionKind
+	expectOperation        v1beta1.Operation
+	expectNamespace        string
+	expectName             string
+	expectObject           bool
+	expectOldObject        bool
+	expectAdmissionRecords bool
```

Comment:
There are so few exceptions, I’d probably just check `admissionExemptResources` directly in the verify method

Reply:
It it would make things clearer, could rename `expect{GVR,Operation,Namespace,Name}` to `record{GVR,Operation,Namespace,Name}`. The other fields (expectGVK, expectOldObject, expectNewObject) are the expectations of anything that is recorded